pkgs/by-name/dr/drip/package.nix

```
@@ -31,16 +35,29 @@ stdenv.mkDerivation rec {
     mkdir $out
     cp ./* $out -r
     wrapProgram $out/bin/drip \
-      --prefix PATH : "${which}/bin"
+      --prefix PATH : ${
+        lib.makeBinPath [
```

```suggestion
        lib.makeBinPath [
          coreutils
```

otherwise the program would always fail when run by the `versionCheckHook` below